### PR TITLE
ZING-7830: add Zenoss OpenCensus exporter

### PR DESCRIPTION
**Pull request by cluther**

### CODE REVIEW

**Comment by cluther** [comment]
retest this please

File: grpc.go [hunk 1]
```diff
@@ -97,68 +111,167 @@ func NewGRPCServer(ctx context.Context, logger, auditLogger *logrus.Entry) *grpc
...
-			"MetricPrefix": "zenoss",
-			"ProjectID":    stackDriverOpts.ProjectID,
-		}
-		logger.WithFields(fields).Info("Staring instrumentation")
-		var err error
-		exporter, err = stackdriver.NewExporter(stackDriverOpts)
 		if err != nil {
-			logger.WithError(err).Fatal("Unable to create trace/metric exporter")
-			return
+			logger.WithError(err).Error("Failed to start Stackdriver exporter")
```
Comment (jplouis) [inline grpc.go]: Old code was fatal if instrumentation could not be started. Seems like it should be the same

File: grpc.go [hunk 2]
```diff
@@ -97,68 +111,167 @@ func NewGRPCServer(ctx context.Context, logger, auditLogger *logrus.Entry) *grpc
...
-			logger.Info("Registering metric exporter")
-			view.RegisterExporter(exporter)
-			interval := 60 * time.Second
-			view.SetReportingPeriod(interval)
-			logger.WithField("ReportingInterval", interval).Info("Registered metric exporter")
+			zenossExporter, err := startZenossExporter(
+				ctx, logger.WithField("opencensus-exporter", "zenoss"))
+
+			if err != nil {
+				logger.WithError(err).Error("Failed to start Zenoss exporter")
```
Comment (jplouis) [inline grpc.go]: Same as above

File: metrics.go [hunk 3]
```diff
@@ -78,3 +128,28 @@ func RecordMillisecondLatency(ctx context.Context, ms *stats.Int64Measure, since
 	ctx, _ = tag.New(ctx, mutator...)
 	stats.Record(ctx, ms.M(SinceInMS(since)))
 }
+
+type grpcStatsHandler struct {
```
Comment (jplouis) [inline metrics.go]: Add a comment about what this grpcstats handler is for so we don't forget like last time

**Pull request merged**